### PR TITLE
Move debrack() to utils.js, remove unused vocab/rdf.js functions

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
The only function from vocab/rdf.js used anywhere was `debrack()`.
Moved it to `utils.js`, removed all the other unused functions in that
file, and the file itself.

### CODE REVIEW

File: lib/vocab/rdf.js [hunk 1]
```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright 2014 IBM Corporation.
```
Comment (nicola) [inline lib/vocab/rdf.js]: I don't know what's up with this

**Comment by nicola** [comment]
LGTM (except the copyright note)

File: lib/vocab/rdf.js (hunk 1 above)
Comment (dmitrizagidulin) [inline lib/vocab/rdf.js]: Lol. wasn't me, didn't do it :)

Comment (dmitrizagidulin) [inline lib/vocab/rdf.js]: (It's fine, though, we're deleting that file anyway.)

**Pull request merged**